src/pages/register/register.js

```diff
@@ -59,11 +57,11 @@ function Register() {
     if (allFieldsValidated) {
       try {
         setLoading(true);
-        await registerUser(user);
+        await registerUser(user, language);
```

Comment:
Don't call async actions in the components as much as possible, please. Move the call of the mutation for user creation to the `user.saga` and here only dispatch an action to trigger an action. In component, we should avoid `awaits` as much as possible

Reply:
removed

Reply:
removed